src/__tests__/a11yAPI.test.js

```diff
@@ -27,13 +27,13 @@ class Button extends React.Component<any> {
       <TouchableOpacity
         accessibilityHint={BUTTON_HINT}
         accessibilityLabel={BUTTON_LABEL}
-        accessibilityRole={BUTTON_ROLE}
+        accessibilityRole={BUTTON_ROLE} // eslint-disable-line react-native-a11y/has-valid-accessibility-role
```

Comment:
BUTTON_ROLE is set to 'button', so maybe it would be better to use it here directly and do not suppress eslint for a11y?